opendebates/views.py

```diff
@@ -186,6 +186,20 @@ def vote(request, id):
         voter.source = request.COOKIES['opendebates.source']
         voter.save()
 
+    if voter.user and voter.user != request.user:
```

Comment:
Okay. I had the hardest time getting my head around the logic here for some reason.